src/Compilers/CSharp/Test/Semantic/Semantics/CollectionLiteralTests.cs

```diff
@@ -6303,33 +6441,5 @@ static async Task<T> F<T>(T t)
                 """;
             CompileAndVerify(new[] { source, s_collectionExtensions }, expectedOutput: "[3, 1, 2, 4], ");
         }
-
-        [ConditionalFact(typeof(CoreClrOnly), AlwaysSkip = "PROTOTYPE: 'IAsyncEnumerable<int>' does not contain a definition for 'GetAwaiter'")]
-        public void Async_03()
```

Comment:
>Async_03 [](http://example.com/codeflow?start=20&length=8)

Added open issue to test plan [https://github.com/dotnet/roslyn/issues/66418](https://github.com/dotnet/roslyn/issues/66418): support `IAsyncEnumerable<T>` for spread element?